pyproject.toml

```diff
@@ -8,13 +8,23 @@
 # by the Apache License, Version 2.0.
 
 [tool.black]
-target_version = ["py311"]
+target_version = ["py310"]
```

Comment:
Did we have to downgrade the version here for some reason?

Reply:
The system Python shipped with the Ubuntu base image is Python 3.10, so using some 3.11-isms were not compatible. I've updated the build image to use Debian 12 as its base, which should ship Python 3.11, so I was able to revert this.